unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/CollectionAddProcedureTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2021 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.block.procedure;
+
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.impl.utility.StringIterate;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class CollectionAddProcedureTest
```

Comment:
Hi Alex, noticed there were few other methods in this class that are not being covered in the test cases. Can you add tests for those as well?